packages/gatsby-plugin-image/src/index.ts

```diff
@@ -25,4 +25,5 @@ export {
   ImageFormat,
   Layout,
   Fit,
+  EVERY_BREAKPOINT,
```

Comment:
Can you describe why this is necessary? I don't see it being used in your PR

Reply:
No not at all anymore :D Can't find any reason, gonna remove it now :)